lib/kube/proxy/forwarder.go

```diff
@@ -804,7 +804,7 @@ func (f *Forwarder) exec(ctx *authContext, w http.ResponseWriter, req *http.Requ
 	}
 	defer proxy.Close()
 
-	f.log.Debugf("Created streams, getting executor.")
+	f.log.Debugf("Getting executor")
```

Comment:
I suggest to remove this log entry, I assume it will be very verbose as it's called every time